src/site/content/en/blog/media-session/index.md

```diff
@@ -476,6 +479,40 @@ navigator.mediaSession.setActionHandler('hangup', () => {
 });
 ```
 
+## Presenting slides actions {: #presenting-slides-actions }
+
+When the user puts their slides presentation into a Picture-in-Picture window,
+the browser may display controls for navigating through slides. When the user
+clicks those, the website handles them through the Media Session API. For an
+example, see the [Presenting Slides sample].
+
+{% Aside %}
```

Comment:
Please use the compat component rather than a note that will go out of date.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/MediaSession/setActionHandler and https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data%20setActionHandler&type=code show you that MDN doesn't have this kind of granularity yet, hence the note for now.

Reply:
It just needs adding to BCD.

Reply:
If MDN has no docs for this please add it to the API docs trix and note that BCD doesn't yet exist.

Reply:
@rachelandrew It's not clear to me if this PR is now ready to be merged or if I need to take some other action.